README.md

```diff
@@ -139,6 +139,7 @@ These themes will be installed in a default installation.
 
 - [bragi](https://github.com/eduNEXT/ednx-saas-themes/tree/edunext/mango.master)
 
+You can now set the default theme you choose. When writing the config.yml file, the defautl theme will be the first at the DISTRO_THEMES_NAME argument.
```

Comment:
With the lasts changes, it should be different. Could say something like "define a DISTRO_DEFAULT_SITE_THEME to set a theme, but if it isn't defined, the default theme will be the first at the DISTRO_THEMES_NAME argument."

Reply:
Fixed @JuanDavidBuitrago